eodag/resources/product_types.yml

```diff
@@ -2414,7 +2414,7 @@ CAMS_GFE_GFAS:
   sensorType: ATMOSPHERIC
   license: proprietary
   title: CAMS global biomass burning emissions based on fire radiative power (GFAS)
-  missionStartDate: "2003-01-01T00:00:00Z"
+  missionStartDate: "2018-07-02T00:00:00Z"
```

Comment:
How did you find this date? When I go to cop ads and use their download UI, the earliest date I can select is in 2020 
![image](https://github.com/user-attachments/assets/aae5aed1-b2d5-449e-8308-34d86c527342)